src/main/webapp/Portal.html

```diff
@@ -93,126 +93,110 @@
       // SWC-5959: Before loading the SRC library, set value so actual column names are shown in Synapse.org
       window.localStorage.setItem('force-display-original-column-names', 'true')
     </script>
-    <script>
-      // determine cdn endpoint (staging or prod, do not use a cdn for local or dev builds)
-      var cdnEndpoint = '//cdn-' + location.hostname + '/'
-      var processEnvironment = 'production'
-      var synapseOrgRegex = /(www|staging|tst)\.synapse\.org$/
-
-      if (!synapseOrgRegex.test(location.hostname)) {
-        cdnEndpoint = '/'
-        processEnvironment = 'development'
-      }
```

Comment:
Logic for CDN moved to the HtmlInjectionFilter. Vite should handle the `processEnvironment` logic in the bundling phase.